libs/barista-components/confirmation-dialog/BUILD.bazel

```diff
@@ -1,7 +1,42 @@
-load("//tools/bazel_rules:index.bzl", "stylelint")
+load("@io_bazel_rules_sass//:defs.bzl", "multi_sass_binary", "sass_library")
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
+load("//tools/bazel_rules:index.bzl", "jest", "ng_module_view_engine", "stylelint")
 
 package(default_visibility = ["//visibility:public"])
 
+ng_module_view_engine(
+    name = "compile",
+    srcs = glob(
+        include = ["**/*.ts"],
+        exclude = [
+            "**/*.spec.ts",
+            "src/test-setup.ts",
+        ],
+    ),
+    angular_assets = [
+        ":styles",
+        ":src/confirmation-dialog.html",
```

Comment:
you are missing the `confirmation-dialog-state.html` here. 

But what you can try is to bazelify the confirmation dialog state as own dependency.
You can DM me for this :)